src/lib/loggers.js

```diff
@@ -61,12 +61,24 @@ loggers.error = function() {
  * apply like other functions do
  */
 function apply(f, args) {
-    if(f.apply) {
-        f.apply(f, args);
+    if(f && f.apply) {
+        try {
+            // `this` should always be console, since here we're always
+            // applying a method of the console object.
+            f.apply(console, args);
```

Comment:
In current browsers this works anyway, but `console` is really the right `this` here and some old browsers (Chrome 51 and presumably before) need it to be correct. See eg https://github.com/plotly/phoenix-integration/issues/223#issuecomment-431863801